demos/select.html

```diff
@@ -195,6 +213,13 @@ <h2 class="mdc-typography--title">CSS Only</h2>
         disabledCb.addEventListener('change', function() {
           select.disabled = disabledCb.checked;
         });
+
+        cssAlternateColors.addEventListener('change', function() {
+          selectCss.classList[cssAlternateColors.checked ? 'add' : 'remove']('demo-select-custom-colors');
+        });
+        cssDisabled.addEventListener('change', function() {
+          cssDisabled.checked ? selectCss.setAttribute('disabled', 'disabled') : selectCss.removeAttribute('disabled');
```

Comment:
Can't this be simplified to `selectCss.disabled = cssDisabled.checked`?

Reply:
No, that doesn't work. The `selectCss` variable is the `div` container that contains the select and the bottom line. A div doesn't have a disabled value, so setting it requires that we add the attribute to the div tag itself. 

Our css styles apply to a disabled selector (`.mdc-select[disabled]`) to "Imitate native disabled functionality".

Reply:
Oh, oops, I'm dumb and was assuming it was the select element itself, and forgot the context.

Reply:
Just to track the conversation here, I discussed this more offline with Will and realized that we've actually got some awkwardness with disabled CSS-only selects, in that you actually need to apply the disabled attribute to both the top-level element (to make it look disabled) _and_ the select (to make it actually behave disabled). Needing to apply the `disabled` attribute to an element for which it has no native functional meaning is really weird, and likely a remnant of select's structure prior to #1097.